meteor/client/ui/Settings/ConfigManifestSettings.tsx

```diff
@@ -80,14 +84,75 @@ function filterLayerMappings(
 	return result
 }
 
+function getFilteredTableValuesFromComparison(
+	targetFullAttribute: string,
+	item: ConfigManifestEntrySelectFromTableEntryWithComparisonMappings<boolean>,
+	configPath: string,
+	showStyleBaseDBObject: ShowStyleBase,
+	showStyleVariantDBObject?: ShowStyleVariant
+): string[] {
+	const sourceAttribute = `${configPath}.${item.sourceTableId}`
+	const targetAttribute = targetFullAttribute.split('.').slice(0, -1).join('.')
```

Comment:
Why `-1`? Then it only slices through the second-to-last element. Is that intentional?

Reply:
Before manipulation: `blueprintConfig.GfxDefaults.0.DefaultDesign`
After manipulation: `blueprintConfig.GfxDefaults.0`
By doing it we get the row rather than a specific value being dealt with.

Reply:
Alternatively we should expect the tableRow attribute path, and then this logic can be moved out to where it is used.

Reply:
This has been updated since resolving merge conflicts showed that the attribute had changed.